docs/deployment/metrics.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Metrics"
+layout: default
+sort: 7
+---
+
+# Metrics
+
+Metrics are configured to be exposed using prometheus.Operator API's by
+default. If you want to expose metrics using the prometheus.Operator
+API's you need to install the prometheus.Operator in your cluster.
+
+The exposed metrics are
+
+| Metric        | Type | Meaning |
+| --------------- | ---------------- | ---------------- |
+|  `nfd_master_build_info`           | Gauge | Version from which nfd-master was built. |
+|  `nfd_worker_build_info`           | Gauge | Version from which nfd-worker was built. |
+|  `nfd_updated_nodes`           |  Counter | Time taken to label a node |
+|  `nfd_crd_processing_time`          |  Gauge | Time taken to process a NodeFeatureRule CRD |
+| `nfd_feature_discovery_duration_seconds` |  HistogramVec | Time taken to discover features on a node |
```

Comment:
nit: there is clearly some formatting in the table, fix it a bit, e.g. (feel free to improve)

```suggestion
| Metric                             | Type    | Meaning |
| ---------------------------------- | ------- | ---------------- |
| `nfd_master_build_info`            | Gauge   | Version from which nfd-master was built. |
| `nfd_worker_build_info`            | Gauge   | Version from which nfd-worker was built. |
| `nfd_updated_nodes`                | Counter | Time taken to label a node |
| `nfd_crd_processing_time`          | Gauge   | Time taken to process a NodeFeatureRule CRD |
| `nfd_feature_discovery_duration_seconds` | HistogramVec | Time taken to discover features on a node |
```